Extensions/blacklist.js

```diff
@@ -738,6 +762,13 @@ XKit.extensions.blacklist = new Object({
 		$(obj).addClass("xblacklist_blacklisted_post");
 		$(obj).prepend(block_excuse);
 
+		$(obj).find('.xblacklist_open_post').each(function() {
```

Comment:
could we do `$(obj).on('click', '.xblacklist_open_post', function() { $(obj).remove(); });`? now that unhiding the post is as simple as removing the excuse container

Reply:
There's also the `$(m_div).removeClass("xblacklist_blacklisted_post");` at least. It doesn't quite work on the old dash but unhide_post is a good candidate for rewriting

Reply:
(assuming this works we could lose all the post id code above too, if im interpreting it right)

Reply:
do we still even need `xblacklist_blacklisted_post` for anything?

Reply:
I've rewritten this code to be less paranoid/verbose

Reply:
On the old dash it's used for styling